blocks/header/header.css

```diff
@@ -289,29 +197,41 @@ header a.button::after {
   margin-left: 200px;
 }
 
-.mega-menu-show {
-  opacity: 1 !important;
+.logo-container {
+  height: 100%;
+  display: flex;
+  justify-content: start;
+  align-items: center;
+  max-width: var(--section-desktop-max-width);
+  margin: 0 auto;
+  padding: 0 var(--body-padding);
 }
 
-.home-solutions-link-hover {
-  color: #FFF;
+header.expanded .logo-container {
+  height: var(--nav-height);
+  position: absolute;
+  top: 0;
+  left: 0;
 }
 
-.home-solutions-link-default {
-  color: #dedede;
+.logo-container > p {
+  display: none;
 }
 
-.header-wrapper {
-  position: relative;
-  z-index: 999;
+.login-buttons a p {
+  text-align: center;
+  font-size: var(--body-font-size-xs);
+  font-weight: 500; 
```

Comment:
This should be defined as a global variable like `--font-weight-bold` and similar for other font-weights.